packages/core-connector-interface/src/types/acknowledgedTicket.d.ts

```diff
@@ -11,7 +11,7 @@ declare interface AcknowledgedTicketStatic {
   ): AcknowledgedTicket
 }
 
-declare interface AcknowledgedTicket {
+declare interface AcknowledgedTicket extends Uint8Array {
```

Comment:
This seems unrelated?

Reply:
Should be included with Node.JS 14 upgrade

Reply:
can you rebase then?